PracticeTestsBackend/app/core/main.py

```
@@ -0,0 +1,53 @@
+# Install pymongo, PyMuPDF, and selenium if you haven't already:
+# pip install pymongo PyMuPDF selenium webdriver-manager
+
+# This script processes PDF files in a specified directory, extracts text and images,
+# and uploads the extracted data to a MongoDB database. It also handles the creation of
+# necessary directories for storing text files and images.
+# It assumes the existence of `extractor`, `jsonparse`, and `upload` scripts for extracting text and images, parsing questions, and uploading to MongoDB.
+
+import os
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)